core/src/main/java/apoc/meta/Meta.java

```diff
@@ -159,7 +159,11 @@ public static Types from(String typeName) {
             }
             typeName = typeName.toUpperCase();
             for (Types type : values()) {
-                if (type.name().startsWith(typeName)) return type;
+                final String name = type.name();
+                // check, e.g. both "LOCAL_DATE_TIME" and "LOCALDATETIME"
+                if (name.startsWith(typeName) || name.replace("_", "").startsWith(typeName)) {
+                    return type;
+                }
```

Comment:
Scusa facciamolo sempre tanto figurati che cambia
```suggestion
                final String name = type.name().replace("_", "");
                // check, e.g. both "LOCAL_DATE_TIME" and "LOCALDATETIME"
                if (name.startsWith(typeName)) {
                    return type;
                }
```